### PR TITLE
BSP-363: extracted reusable values into consts

### PR DESCRIPTION
**Pull request by yarpo**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BSP-363

### Change description ###
Extracted reusable strings into consts

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/bsp/common/mapper/AddressMapper.java [hunk 1]
```diff
@@ -27,15 +46,15 @@ public static void applyMappings(
...
-        address.put(nestedFieldPrefix + "County", "County");
-        address.put(nestedFieldPrefix + "Postcode", "PostCode");
-        address.put(nestedFieldPrefix + "Town", "PostTown");
-        address.put(nestedFieldPrefix + "Country", "Country");
+        address.put(nestedFieldPrefix + OcrSuffixes.LINE_1, Fields.LINE_1);
+        address.put(nestedFieldPrefix + OcrSuffixes.LINE_2, Fields.LINE_2);
+        address.put(nestedFieldPrefix + OcrSuffixes.POSTCODE, Fields.POSTCODE);
+        address.put(nestedFieldPrefix + OcrSuffixes.TOWN, Fields.TOWN);
+        address.put(nestedFieldPrefix + OcrSuffixes.COUNTY, Fields.COUNTY);
+        address.put(nestedFieldPrefix + OcrSuffixes.COUNTRY, Fields.COUNTRY);
```
Comment (jakub-kozlowski) [inline src/main/java/uk/gov/hmcts/reform/bsp/common/mapper/AddressMapper.java:57]: `OcrSuffix` would read better than the plural form?

Comment (jacmurphy) [reply src/main/java/uk/gov/hmcts/reform/bsp/common/mapper/AddressMapper.java:57]: I agree

File: src/main/java/uk/gov/hmcts/reform/bsp/common/mapper/AddressMapper.java [hunk 2]
```diff
@@ -12,6 +12,25 @@
 
 public class AddressMapper {
 
+    public static class Fields {
```
Comment (jvictorcf) [inline src/main/java/uk/gov/hmcts/reform/bsp/common/mapper/AddressMapper.java:15]: I think this class doesn't need to be static. Happy to keep it static if you prefer that, though.

File: src/main/java/uk/gov/hmcts/reform/bsp/common/mapper/AddressMapper.java [hunk 3]
```diff
@@ -12,6 +12,25 @@
...
+        public static final String LINE_1 = "AddressLine1";
+        public static final String LINE_2 = "AddressLine2";
+        public static final String POSTCODE = "PostCode";
+        public static final String TOWN = "PostTown";
+        public static final String COUNTY = "County";
+        public static final String COUNTRY = "Country";
+    }
+
+    public static class OcrSuffixes {
```
Comment (jvictorcf) [inline src/main/java/uk/gov/hmcts/reform/bsp/common/mapper/AddressMapper.java:24]: ditto

**Pull request merged**